docs/wiki/End-user-Guide/preparing-your-modpack.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Preparing your modpack
+---
+
+# Preparing Your Modpack
+
+## Overview
+
+Cleanroom Loader is compatible with the vast majority of Forge mods and will usually run them without issue.  
+However, some mods remain **deprecated, redundant, or outright incompatible**—either because they’ve been superseded by better alternatives or because they introduce critical problems.
+
+To prepare your modpack for Cleanroom Loader, review the lists below and make replacements where necessary.
+
+---
+
+## Deprecated / Incompatible Mods
```

Comment:
since this is intended to be a basic list for "make it run on cleanroom", i would suggest moving any of the mods that are not strictly incompatible or directly redundant to another location.